packages/devtools-launchpad/src/components/LandingPage.css

```diff
@@ -84,14 +84,19 @@
 
 .landing-page .panel header input {
   flex: 1;
-  color: var(--theme-body-color);
+  background-color: var(--theme-body-background);
+  color: var(--theme-comment);
```

Comment:
I'm afraid this color looks too similar to the one in the placeholder 

![capture du 2017-01-06 08-15-31](https://cloud.githubusercontent.com/assets/578107/21710393/94e04a6e-d3e8-11e6-8c4e-f3f2ba3464a3.png)

Can we try something more contrasting ?